example/calculate_pi/calculate_pi.cpp

```diff
@@ -71,12 +71,29 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
   Metadata m({Metadata::Cell, Metadata::Derived});
   package->AddField(field_name, m, DerivedOwnership::unique);
 
+  // Add a named MeshPack by registering a function that packs it
+  package->AddMeshBlockPack("in_or_out", [](Mesh *pmesh) {
+    // 1 means pack is 1 meshblock, <1 means use entire mesh
+    int pack_size = pmesh->DefaultPackSize();
```

Comment:
This seems unnecessarily complicated.  Why not just have `pmesh->DefaultPackSize()` return `block_list.size()` instead of something < 1?

Reply:
Because I didn't think of it. Yes, I can make that change. In `parthinput`, `<1` needs to be an option since I don't want the user to have to know what sizes are available. But I agree here.

Reply:
Good point. Resolved.